flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinition.java

```diff
@@ -90,6 +110,18 @@ public boolean hasRuntimeImplementation() {
         return isRuntimeProvided || runtimeClass != null;
     }
 
+    public boolean isInternal() {
+        return isInternal;
+    }
+
+    public String getQualifiedName() {
```

Comment:
the term "qualified name" IMO is misleading, as it can be easily confused with FQCN from Java terminology. Just identifier?

Reply:
+1 for me

Reply:
Hm, rethought of that, qualifier, qualified name makes sense for SQL terms.